cogs/actor.py

```diff
@@ -61,20 +61,16 @@ def _save_reactions(self):
 
     def _check_match_string(self, match: str):
         """Check if match string is valid"""
-        return match[:1].upper() in ["F", "A", "S", "E"]
+        return match[:1].upper() in ["full", "any", "starts", "ends"]
 
     def _check_filename_extension(self, filename: str):
-        return filename.split(".")[-1] in ["jpg", "jpeg", "png", "webm", "mp4", "gif"]
+        return filename.split(".") in ["jpg", "jpeg", "png", "webm", "mp4", "gif"]
```

Comment:
```suggestion
        return filename.split(".")[-1] in ["jpg", "jpeg", "png", "webm", "mp4", "gif"]
```
This was a mistake made because of converting from letter to word reaction indicators. This is a list and we want the extension. The original was correct.